src/renderer/angular/controllers/commons/dialogs/terms-controller.js

```diff
@@ -41,8 +41,9 @@ function TermsDialogController($rootScope, $scope, $log, $q, $mdDialog, SqlLiteS
 
 	$scope.scrollToEndContainer = direction => {
 		if (direction === 'bottom') {
-			$scope.scrolledBottom = true;
-			$scope.$apply();
+			$timeout(() => {
```

Comment:
What does this timeout do, @sk91?

Reply:
@dfcowell it does scope.$apply in a safe way ($scope.apply will fail if digest cycle is in progress)

Popular prectice in angular